src/main/java/com/attacktimer/AttackTimerMetronomeTileOverlay.java

```diff
@@ -78,21 +83,45 @@ public Dimension render(Graphics2D graphics)
             {
                 graphics.setFont(new Font(FontManager.getRunescapeFont().getName(), Font.PLAIN, config.fontSize()));
             }
+            else if (config.fontType() == FontTypes.BOLD)
+            {
+                graphics.setFont(new Font(config.fontType().toString(), Font.BOLD, config.fontSize()));
+            }
             else
             {
                 graphics.setFont(new Font(config.fontType().toString(), Font.PLAIN, config.fontSize()));
             }
 
-            final int height = client.getLocalPlayer().getLogicalHeight()+20;
-            final LocalPoint localLocation = client.getLocalPlayer().getLocalLocation();
-            final Point playerPoint = Perspective.localToCanvas(client, localLocation, client.getTopLevelWorldView().getPlane(), height);
-
             // Countdown ticks instead of up.
             // plugin.tickCounter => ticksRemaining
             int ticksRemaining = plugin.getTicksUntilNextAttack();
-            OverlayUtil.renderTextLocation(graphics, playerPoint, String.valueOf(ticksRemaining), config.NumberColor());
-        }
 
+            switch (config.ticksPosition()) {
+                case TOP:
+                    final LocalPoint localLocationT = client.getLocalPlayer().getLocalLocation();
+                    final Point playerPointT = Perspective.localToCanvas(client, localLocationT, client.getTopLevelWorldView().getPlane(), 214 + config.heightTickOffset());
+                    OverlayUtil.renderTextLocation(graphics, playerPointT, String.valueOf(ticksRemaining), ticksRemaining == 1 ? config.LastColor() : config.NumberColor());
+                    break;
+
+                case CENTERED:
+                    final LocalPoint localLocationC = client.getLocalPlayer().getLocalLocation();
+                    final Point playerPointC = Perspective.localToCanvas(client, localLocationC, client.getTopLevelWorldView().getPlane(), 100 + config.heightTickOffset());
+                    OverlayUtil.renderTextLocation(graphics, playerPointC, String.valueOf(ticksRemaining), ticksRemaining == 1 ? config.LastColor() : config.NumberColor());
+                    break;
+
+                case BOTTOM:
+                    final Point playerPointB = player.getCanvasTextLocation(graphics, String.valueOf(ticksRemaining), 10 + config.heightTickOffset());
+                    OverlayUtil.renderTextLocation(graphics, playerPointB, String.valueOf(ticksRemaining), ticksRemaining == 1 ? config.LastColor() : config.NumberColor());
+                    break;
+
+                case DEFAULT:
+                    final int height = client.getLocalPlayer().getLogicalHeight() + 20 + config.heightTickOffset();
```

Comment:
Huh, I guess height offset never worked in the first place. Thanks for fixing it.